packages/@webex/plugin-cc/src/WebSocket.ts

```diff
@@ -0,0 +1,182 @@
+import Mercury from '@webex/internal-plugin-mercury';
+
+export const webSocketConfig = {
```

Comment:
Ideally, this config should be present in separate file

Reply:
keeping it simple to avoid multiple configs in cc plugin.